dj_beneficiary/models.py

```diff
@@ -165,20 +154,17 @@ def __str__(self):
     ("Female", _("Female"))
 )
 
-class Agent(models.Model):
+class BaseAgentAbstract(models.Model):
```

Comment:
we dont need this as abstruct 
I dont remember asking you to make this abstruct

Reply:
Okay